zilliqa/src/api/types/eth.rs

```diff
@@ -422,7 +422,7 @@ pub struct CallParams {
     pub gas_price: Option<U128>,
     #[serde(default)]
     pub value: U128,
-    #[serde(default, deserialize_with = "deserialize_data")]
+    #[serde(default, deserialize_with = "deserialize_data", alias = "input")]
```

Comment:
Did you do this?

Reply:
Will separate this to another PR. I'd rather leave it as-is here if possible because this change would affect transactions in general. A subsequent PR will resolve this and I'll test separately.

Reply:
I made this change in #1534 because I think its important it gets a separate changelog entry rather than ending up being merged with a bunch of UCCB stuff.

Reply:
Please can you review #1534 and merge main into this PR once it is merged.